src/system.ts

```diff
@@ -27,6 +27,14 @@ export interface SystemStave {
   debugNoteMetrics?: boolean;
 }
 
+interface StaveInfo {
```

Comment:
I don't understand why we need this separately? Why not use `SystemStave` above?

Reply:
I left `SystemStave` so as not to modify the interface of `System.addStave`. However the relation `voices` to `staves` is not N to 1, one `Voice` can have `Notes` in different `Staves`, as a consequence, they must be sotred separately to allow the N to M relation. Then `SatveInfo` is only used in `System.format` for this reason I placed it apart. This also allows to return `this.partStaves` directly in `System.getStaves()`.